lib/js/components/Buttons/IconButton/IconButton.vue

```diff
@@ -43,7 +44,13 @@
 			:elevation="elevation"
 			:color="isButtonColor ? color : null"
 		>
-			<wnl-icon class="a-iconButton__icon" :icon="icon" :size="iconSize" />
+			<wnl-icon
+				v-if="state === ICON_BUTTON_STATES.LOADING"
+				class="a-iconButton__icon"
+				:icon="ICONS.FA_SPINNER_THIRD"
+				:size="iconSize"
+			/>
+			<wnl-icon v-else class="a-iconButton__icon" :icon="icon" :size="iconSize" />
```

Comment:
the only difference in this `v-if` is icon. Why don't render other icon conditionally in `:icon` prop?

Reply:
Now is the second difference - `spinning` - I think this condition should stay.